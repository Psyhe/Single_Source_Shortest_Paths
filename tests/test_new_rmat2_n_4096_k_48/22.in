4096 1886 1970
1886 350 201
1886 1628 178
1886 2910 115
1886 3934 36
1886 1758 52
1887 3935 103
1887 1055 103
1887 1867 160
1888 1376 137
1888 1792 11
1888 1344 97
1888 1808 101
1888 817 210
1888 1880 98
1888 3620 158
1888 1382 122
1888 3915 223
1888 3938 65
1888 1986 98
1888 1632 147
1888 1772 60
1888 1698 56
1888 324 112
1888 1581 81
1888 1904 59
1888 1896 125
1888 612 228
1888 1377 239
1888 1106 35
1889 1903 96
1889 1636 33
1889 1859 1
1889 1377 146
1889 2869 43
1889 549 132
1889 1860 6
1889 2025 77
1889 867 79
1889 3424 103
1889 864 66
1889 832 118
1889 1793 5
1889 323 59
1889 584 13
1889 3361 207
1890 358 140
1890 880 211
1890 2807 146
1890 1826 0
1890 1830 146
1890 3270 191
1890 1362 97
1890 2016 172
1890 874 65
1890 2018 11
1891 1991 13
1891 553 149
1891 3587 121
1891 3235 131
1892 1344 93
1892 3874 124
1892 1877 52
1892 868 161
1892 1856 224
1892 1397 118
1892 3332 8
1893 1957 59
1893 1668 107
1893 68 153
1893 1844 183
1893 1897 176
1893 2881 49
1893 3824 207
1893 1121 33
1893 868 83
1893 2023 64
1893 865 34
1893 877 178
1893 1860 206
1893 3697 41
1893 117 72
1893 1879 172
1894 3654 219
1894 3910 53
1894 1230 144
1894 871 122
1894 1830 203
1895 1829 224
1895 1999 13
1895 870 133
1895 1611 205
1895 1919 131
1895 1862 248
1895 1379 93
1896 2036 47
1896 1888 125
1896 2648 81
1896 846 192
1896 3912 28
1896 3434 37
1896 1832 51
1896 1368 145
1896 2026 15
1896 1786 203
1896 2042 122
1896 877 39
1896 3440 63
1896 1900 97
1896 1376 164
1896 2664 172
1896 1032 239
1896 3936 214
1897 1656 112
1897 2024 187
1897 1893 176
1897 3501 203
1897 877 126
1897 2017 21
1897 3683 2
1897 4012 40
1898 1323 229
1898 3436 45
1898 2043 22
1898 1612 126
1898 1953 59
1899 1642 231
1899 1121 92
1899 1115 56
1899 1347 85
1899 1390 91
1899 2089 23
1899 1646 226
1899 1251 45
1899 1131 197
1900 846 115
1900 1704 110
1900 1644 132
1900 1896 97
1900 1388 52
1901 1624 32
1901 3878 115
1901 871 69
1901 1407 181
1901 1913 190
1901 1389 147
1901 39 20
1901 46 192
1901 812 195
1902 850 16
1902 4074 108
1902 3686 144
1902 620 10
1903 1889 96
1903 1583 81
1903 1402 143
1904 1848 200
1904 1392 90
1904 1944 140
1904 1905 218
1904 1100 169
1904 1888 59
1904 3920 29
1904 1872 250
1904 1778 164
1904 4089 231
1904 552 152
1904 1913 69
1904 1968 191
1904 368 76
1904 1880 178
1904 802 202
1904 1264 156
1904 1361 93
1905 1393 228
1905 344 230
1905 885 184
1905 1969 237
1905 1377 220
1905 4081 104
1905 1904 218
1905 97 136
1905 633 243
1906 2002 227
1906 2034 3
1906 1392 114
1906 890 135
1906 882 43
1906 1504 220
1907 1874 216
1907 883 191
1907 2359 248
1907 1873 252
1907 2595 233
1907 1577 109
1907 1654 45
1907 51 126
1907 3859 178
1907 1651 56
1907 819 25
1907 3891 170
1907 1648 233
1907 1003 46
1908 1844 204
1908 576 190
1908 1876 13
1908 1138 27
1908 1524 141
1908 1136 206
1909 885 160
1909 1917 141
1909 1829 124
1909 1589 30
1909 1852 26
1909 1651 97
1909 865 51
1909 1911 229
1909 1653 135
1910 1380 114
1910 1846 242
1910 614 112
1910 1380 133
1911 2044 61
1911 623 51
1911 3943 73
1911 1367 137
1911 549 154
1911 1909 229
1912 1850 20
1912 1642 50
1912 1919 127
1912 3964 224
1912 1640 119
1912 1144 193
1912 600 41
1913 1649 171
1913 1946 241
1913 1904 69
1913 1901 190
1913 2365 84
1913 1915 186
1914 1915 134
1914 3818 215
1914 890 68
1914 3802 97
1914 2938 115
1914 890 178
1914 1394 183
1915 1914 134
1915 1457 158
1915 382 131
1915 1913 186
1915 795 222
1915 3454 30
1916 1917 246
1916 1820 130
1916 1844 209
1917 2669 9
1917 861 4
1917 1916 246
1917 1909 141
1917 1149 104
1918 2014 203
1918 1870 206
1919 1912 127
1919 1895 131
1919 1647 151
1919 1387 186
1919 2013 4
1919 2823 44
1920 512 104
1920 3968 56
1920 4088 167
1920 3850 165
1920 4040 130
1920 1489 194
1920 1952 117
1920 4064 212
1920 900 44
1920 2962 54
1920 1738 156
1920 2465 172
1920 1795 247
1920 1947 44
1920 1921 30
1920 3876 92
1920 912 135
1920 900 115
1921 1667 238
1921 1920 30
1921 1297 8
1921 1188 12
1921 3968 163
1921 773 250
1921 3968 232
1921 3969 114
1921 641 145
1921 428 131
1921 1929 183
1921 425 69
1921 1859 224
1921 3720 120
1922 4038 43
1922 902 105
1922 1936 232
1922 3459 243
1922 2630 40
1922 1938 226
1922 1794 130
1922 1222 119
1922 2842 169
1922 896 82
1922 1666 246
1922 129 171
1922 1474 101
1922 1954 220
1922 1546 64
1922 1410 203
1922 129 238
1922 1506 19
1922 3464 29
1923 529 79
1923 1667 131
1923 897 56
1923 1408 57
1923 1667 181
1923 609 70
1923 2823 151
1923 1665 214
1923 403 137
1923 1409 170
1923 1155 141
1923 1803 242
1923 899 27
1923 1795 72
1923 1408 3
1923 1282 169
1924 1408 231
1924 1933 93
1924 1284 90
1924 3892 68
1924 2180 19
1924 3781 169
1924 1758 71
1924 2022 99
1924 1925 154
1924 4004 87
1924 3781 72
1924 1940 155
1924 981 160
1924 1956 67
1924 1926 83
1924 1668 212
1924 1959 186
1924 1796 121
1924 709 165
1924 3300 110
1924 1324 212
1925 1940 187
1925 1445 163
1925 1924 154
1925 705 99
1925 897 196
1925 1537 52
1925 901 171
1925 1749 189
1925 1437 9
1925 796 129
1925 525 14
1925 2949 198
1925 1949 82
1925 1545 156
1925 1681 248
1926 1670 238
1926 1986 21
1926 3983 53
1926 1924 83
1926 1298 23
1926 1486 219
1926 3970 194
1926 3456 250
1926 3842 164
1926 4006 63
1926 1280 112
1926 1990 14
1926 1670 127
1926 1959 161
1927 1668 168
1927 644 24
1927 1667 204
1927 1674 33
1927 935 109
1927 3815 7
1927 1287 56
1927 4086 62
1927 1411 141
1928 1984 170
1928 1688 158
1928 1666 25
1928 1930 179
1928 1998 49
1928 1874 119
1928 1044 203
1928 2574 157
1928 1224 91
1928 3033 90
1928 1176 185
1928 1674 92
1928 1993 12
1928 1994 71
1928 3976 69
1929 3469 145
1929 457 37
1929 1673 211
1929 1825 27
1929 961 212
1929 1932 27
1929 3465 150
1929 905 152
1929 1945 153
1929 652 11
1929 1673 14
1929 1936 87
1929 1482 231
1929 1559 134
1929 1545 139
1929 385 221
1929 1921 183
1930 392 143
1930 3342 217
1930 1794 2
1930 1928 179
1930 1194 188
1930 394 126
1930 1298 219
1930 3402 140
1930 1416 180
1930 1816 153
1930 1482 43
1930 2026 204
1930 1935 116
1930 910 216
1931 1803 201
1931 1710 149
1931 4011 120
1931 779 80
1931 781 99
1931 3979 168
1932 3485 100
1932 3976 138
1932 1420 53
1932 1708 177
1932 1948 79
1932 1676 193
1932 1668 219
1932 1929 27
1933 3585 160
1933 1421 2
1933 904 193
1933 1740 2
1933 3983 200
1933 933 72
1933 909 250
1933 1924 93
1934 911 136
1934 1951 125
1934 1995 210
1934 1946 160
1934 1806 22
1935 966 112
1935 1679 183
1935 1930 116
1936 912 127
1936 1929 87
1936 1845 182
1936 1986 198
1936 464 206
1936 2992 195
1936 1952 125
1936 3750 4
1936 1682 130
1936 3600 20
1936 272 115
1936 1922 232
1936 1976 208
1936 1680 12
1936 3728 75
1936 3348 98
1936 902 7
1936 1425 20
1936 282 100
1936 1682 30
1936 1856 243
1936 916 215
1937 273 121
1937 944 119
1937 1873 212
1937 656 209
1937 1961 106
1937 3922 150
1937 925 171
1937 1809 139
1937 1793 242
1937 915 147
1937 3888 215
1937 1425 219
1937 401 16
1937 3985 59
1937 1809 181
1937 1939 197
1938 1554 131
1938 1922 226
1938 920 6
1938 1970 195
1938 1302 113
1938 1554 48
1938 1682 205
1938 1362 188
1938 1536 71
1938 1942 62
1939 1971 121
1939 29 94
1939 1411 14
1939 922 71
1939 1937 197
1939 1811 120
1940 2452 60
1940 3360 158
1940 3612 178
1940 1428 24
1940 1415 78
1940 1845 244
1940 3793 188
1940 1925 187
1940 1924 155
1940 1820 60
1940 4052 186
1940 1012 104
1940 1680 231
1940 1433 118
1940 764 189
1940 1303 38
1941 789 98
1941 1649 143
1941 1557 148
1941 1969 72
1941 661 224
1941 3103 152
1941 916 1
1941 1557 146
1942 1676 27
1942 3342 132
1942 1170 220
1942 3891 0
1942 1938 62
1942 702 161
1943 931 124
1943 918 128
1943 2005 12
1943 1415 4
1943 1302 198
1943 3713 99
1943 769 184
1943 1431 185
1944 3480 180
1944 1976 54
1944 2828 205
1944 1904 140
1944 1801 159
1944 3976 105
1944 1439 160
1944 906 125
1945 249 42
1945 1793 44
1945 1179 239
1945 3993 21
1945 1819 6
1945 1929 153
1945 1865 71
1945 781 165
1946 3848 129
1946 1852 94
1946 1913 241
1946 3994 107
1946 1934 160
1946 1977 109
1946 1182 116
1946 907 72
1946 3768 41
1946 1690 77
1946 1554 19
1947 1627 150
1947 3593 97
1947 1690 175
1947 3733 204
1947 1920 44
1947 787 11
1947 791 255
1947 1055 161
1948 1684 70
1948 798 81
1948 1932 79
1948 1694 97
1948 932 95
1948 1725 16
1948 669 59
1948 134 124
1948 1182 69
1948 3724 17
1948 794 179
1948 1724 225
1949 1925 82
1949 36 108
1949 1692 22
1949 2013 70
1949 1435 255
1949 1785 7
1950 1806 219
1950 3466 46
1950 1630 79
1950 314 54
1950 410 221
1950 2006 222
1951 2445 51
1951 1689 147
1951 1934 125
1951 1959 70
1951 3995 170
1951 95 133
1952 1808 101
1952 2946 106
1952 488 161
1952 1824 41
1952 160 196
1952 3493 142
1952 1301 190
1952 1936 125
1952 1057 156
1952 1953 195
1952 4066 25
1952 1968 46
1952 4020 191
1952 1955 54
1952 1920 117
1952 2016 83
1952 928 196
1952 420 34
1952 48 221
1952 1442 108
1952 1760 124
1952 3336 215
1952 1441 164
1952 4001 207
1952 1960 42
1952 1954 64
1953 673 63
1953 1985 25
1953 755 167
1953 1952 195
1953 1840 62
1953 1969 139
1953 1704 195
1953 947 187
1953 1697 49
1953 1072 50
1953 1421 163
1953 1825 171
1953 1898 59
1954 672 171
1954 2018 90
1954 803 30
1954 3498 221
1954 1834 237
1954 1952 64
1954 1960 92
1954 1160 50
1954 4002 181
1954 1922 220
1954 1384 64
1954 995 255
1954 1827 195
1954 1810 207
1954 163 203
1954 1723 161
1954 1827 25
1954 929 164
1954 138 66
1954 1443 60
1955 3971 236
1955 803 144
1955 1824 66
1955 1767 45
1955 1696 179
1955 931 184
1955 1952 54
1955 519 249
1956 4008 3
1956 308 123
1956 1702 183
1956 1668 97
1956 1974 46
1956 292 1
1956 1444 40
1956 1828 142
1956 3716 190
1956 3210 87
1956 1188 229
1956 180 205
1956 3232 33
1956 2030 5
1956 1524 243
1956 996 218
1956 1924 67
1957 672 22
1957 1991 48
1957 813 122
1957 1837 114
1957 1461 215
1957 1893 59
1958 1974 102
1958 3374 135
1958 3744 211
1958 1974 11
1958 1734 220
1959 1346 130
1959 1924 186
1959 1798 145
1959 1951 70
1959 1487 226
1959 1926 161
1959 1699 91
1960 4008 67
1960 2018 129
1960 1836 127
1960 1954 92
1960 1762 227
1960 872 81
1960 1952 42
1960 1580 172
1960 1976 27
1960 673 247
1960 768 8
1960 650 7
1960 653 213
1961 1161 203
1961 1593 183
1961 939 210
1961 936 34
1961 396 80
1961 1937 106
1961 1577 243
1961 169 255
1961 905 207
1962 1574 147
1962 1704 254
1962 954 114
1962 1426 146
1963 1455 163
1963 1451 216
1964 1797 213
1964 1966 5
1964 1572 35
1964 1187 235
1964 2594 161
1965 1725 189
1966 3068 20
1966 3982 216
1966 3116 27
1966 1964 5
1966 3490 17
1967 959 105
1967 1327 204
1967 1516 123
1968 2481 251
1968 1330 236
1968 802 191
1968 3511 36
1968 1424 6
1968 2673 104
1968 696 243
1968 3617 174
1968 1008 44
1968 1696 134
1968 4000 185
1968 1200 216
1968 1904 191
1968 1504 54
1968 954 218
1968 1952 46
1968 1972 190
1968 1712 27
1968 1840 192
1969 3602 28
1969 1465 211
1969 2980 154
1969 1585 133
1969 3249 44
1969 1201 184
1969 2977 19
1969 1841 158
1969 1941 72
1969 3425 250
1969 1953 139
1969 1857 102
1969 1713 246
1969 1905 237
1969 688 82
1970 1470 192
1970 946 68
1970 1938 195
1970 1978 221
1970 1975 57
1970 110 191
1970 1826 240
1970 1712 91
1970 1458 209
1970 50 16
1970 1680 221
1970 1842 135
