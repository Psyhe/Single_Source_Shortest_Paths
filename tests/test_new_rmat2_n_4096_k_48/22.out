159
137
59
50
56
105
18
48
41
176
56
149
51
67
99
114
43
57
51
50
36
68
99
88
54
219
92
52
76
107
140
139
62
252
29
39
60
123
92
96
28
27
60
30
46
166
69
109
90
100
42
62
78
58
94
141
86
125
56
48
42
64
71
56
114
107
62
66
49
102
33
45
114
32
64
55
149
74
38
70
112
69
117
64
79
